.github/workflows/windows-ci.yml

```diff
@@ -2,30 +2,67 @@ name: Windows CI
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
 
 jobs:
   build:
-    runs-on: windows-2019
+    runs-on: windows-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: '3.x' 
-      - uses: BSFishy/pip-action@v1
-        with:
-          packages: conan
-        
-      - name: Conan Install
-        run: conan install . --build missing
-
-      - name: Install Hsm
-        run: |
-         cmake . -DCMAKE_BUILD_TYPE=DEBUG
-         cmake --build . --target install
-
-      - name: Run Hsm Tests
-        run: ctest -VV -C Debug
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - uses: BSFishy/pip-action@v1
+      with:
+        packages: conan
+
+    - name: Cache Conan
+      uses: actions/cache@v2
+      with:
+        path: .conan
+        key: conan-cache-${{ hashFiles('conanfile.txt') }}-windows
+
+    - run: mkdir -p build/dependencies/conan
+
+    - name: Install Conan Dependencies
+      working-directory: build/dependencies/conan
+      shell: bash
+      env: { CONAN_USER_HOME: '${{ github.workspace }}/.conan' }
+      run: conan install ../../.. --build missing
+
+    - name: Cache GTest
```

Comment:
Why not install gtest from conan now?

Reply:
I don't really use conan myself, so I'm not aware of gtest existing on conan. The entirety of gtest was in this repo, so I thought it's just easier (for me) to do it directly.